cmd/kwild/config/flags.go

```diff
@@ -24,6 +24,8 @@ func AddConfigFlags(flagSet *pflag.FlagSet, cfg *KwildConfig) {
 	flagSet.StringVar(&cfg.AppCfg.TLSCertFile, "app.tls-cert-file", cfg.AppCfg.TLSCertFile, "TLS certificate file path for the admin and consensus RPC server (optional)")
 	flagSet.StringVar(&cfg.AppCfg.TLSKeyFile, "app.tls-key-file", cfg.AppCfg.TLSKeyFile, "TLS key file path for the admin and consensus RPC servers (optional)")
 	flagSet.StringVar(&cfg.AppCfg.Hostname, "app.hostname", cfg.AppCfg.Hostname, "kwild Server hostname")
+	flagSet.BoolVar(&cfg.AppCfg.AutheticateRPCs, "app.authenticate-rpcs", cfg.AppCfg.AutheticateRPCs, "Authenticate JSON-RPC requests using the private key")
```

Comment:
What does "using the private key" mean here?  This is a `kwild` flag, but I think you're referring to the client using their private key to create a signature to include in their requests.  For this `kwild` flag, I think the description should be more like "Authenticate call and query RPC request, requiring clients to include a signature in their requests."